test/views/html/DateOfLeavingSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
is it documented somewhere for us to remove this page after all updates have been done before the final merge & release to production?

Reply:
on list for clean up